include/wx/bannerwindow.h

```diff
@@ -14,11 +14,11 @@
 
 #if wxUSE_BANNERWINDOW
 
-#include "wx/bitmap.h"
+#include "wx/bmpbndl.h"
 #include "wx/event.h"
 #include "wx/window.h"
 
-class WXDLLIMPEXP_FWD_CORE wxBitmap;
+class WXDLLIMPEXP_FWD_CORE wxBitmapBundle;
```

Comment:
```suggestion
```
This is unnecessary (just as the old forward declaration was, but as we're changing this anyhow, it's better to remove it instead of preserving).